Toolchain/BuildPython.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
+<<<<<<< HEAD
```

Comment:
Whoops, looks like an unresolved merge conflict snuck in. I also noticed there's a merge commit in your commit list -- we like to follow a rebase + squash workflow so that commits can be directly placed in master from each PR.  https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy

Reply:
Good catch!